config/clients/js/template/example/example1/package.json.mustache

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "example1",
+  "private": "true",
+  "version": "1.0.0",
+  "description": "A bare bones example of using the OpenFGA SDK",
+  "author": "OpenFGA",
+  "license": "Apache-2.0",
+  "scripts": {
+    "run-example": "node example1.mjs"
```

Comment:
Isn't calling it `run` or `start` more common?

Reply:
This was more personal opinion thing, `npm run run` felt odd to have, `npm run start`/`npm start` also felt wrong as it tends to be more "start my server".

But I think you're right that we should use something common so I'll make it `npm run start`